src/main/resources/templates/springboot/rest/build.gradle.tmpl

```diff
@@ -14,15 +14,7 @@ apply plugin: "blackbaud-templates"
 apply plugin: "spring-boot"
 
 dependencies {
-    compile "org.springframework.boot:spring-boot-starter-web"
-    compile "org.springframework.boot:spring-boot-starter-actuator"
-    compile "org.springframework.boot:spring-boot-starter-jersey"
-
-    compile "com.google.guava:guava:18.0"
-    compile "org.projectlombok:lombok:1.16.4"
-    compile("net.sf.dozer:dozer:5.5.1") {
-        exclude group: 'org.slf4j'
-    }
+    compile "com.blackbaud:common-spring-boot-rest:\${springBootVersion}-0+"
```

Comment:
good catch. it's kind of unfortunate that each time we bump the version, we'll need to bump the template since we only bump the version when it's backwards compatible